bundles/org.openhab.binding.smarther/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,141 @@
+portal); OFF = for each module connected to this bridge, status+sensors data are requested to Smarther API gateway on a periodical basis and whenever new
+settings are applied (period can be changed via module's "Status Refresh Period" parameter).</description>
+			<required>false</required>
+			<advanced>true</advanced>
+			<default>true</default>
+		</parameter>
+
+		<parameter name="statusRefreshPeriod" groupName="advancedset" type="integer" min="1" unit="min">
+			<label>Bridge Status Refresh Period (minutes)</label>
+			<description>This is the frequency the Smarther API gateway is called to update bridge status. There are limits to the number of
+requests that can be sent to the Smarther API gateway. The more often you poll, the faster locations are updated - at the risk of running out of your request
+quota.</description>
+			<required>false</required>
+			<advanced>true</advanced>
+			<unitLabel>Minutes</unitLabel>
+			<default>1440</default>
+		</parameter>
+
+	</config-description>
+
+	<!-- Config for Smarther Module -->
+	<config-description uri="thing-type:smarther:module">
+
+		<!-- Parameter groups -->
+		<parameter-group name="topology">
+			<label>Module Topology</label>
+			<description>Reference to uniquely identify the module towards the BTicino/Legrand API gateway.</description>
+		</parameter-group>
+
+		<parameter-group name="advancedset">
+			<label>Advanced Settings</label>
+			<description>Advanced settings of this module.</description>
+		</parameter-group>
+
+		<!-- Parameters -->
+		<parameter name="plantId" groupName="topology" type="text" pattern="[0-9a-f]{8}[-]([0-9a-f]{4}[-]){3}[0-9a-f]{12}">
+			<label>Location Plant Id</label>
+			<description>This is the Plant Id of the location the Chronothermostat module is installed in, provided by Smarther API.</description>
+			<required>true</required>
+		</parameter>
+
+		<parameter name="moduleId" groupName="topology" type="text" pattern="[0-9a-f]{8}[-]([0-9a-f]{4}[-]){3}[0-9a-f]{12}">
+			<label>Chronothermostat Module Id</label>
+			<description>This is the Module Id of the Chronothermostat module, provided by Smarther API.</description>
+			<required>true</required>
+		</parameter>
+
+		<parameter name="settingsAutoupdate" groupName="advancedset" type="boolean">
+			<label>Module Settings Auto-Update</label>
+			<description>ON = the module settings are automatically updated according to the module status whenever it changes (e.g. polling, notification,
+etc.). OFF = the module settings are aligned to the module status only upon module initialization.</description>
+			<required>false</required>
+			<advanced>true</advanced>
+			<default>false</default>
+		</parameter>
+
+		<parameter name="programsRefreshPeriod" groupName="advancedset" type="integer" min="1" unit="min">
+			<label>Programs Refresh Period (hours)</label>
```

Comment:
unit attribute might be wrong.

Reply:
Indeed, checking the xsd it turned out the correct one to be `h`.